packages/style-engine/README.md

```diff
@@ -194,13 +194,13 @@ in your code._
 
 <!-- START TOKEN(Autogenerated API docs) -->
 
-### generate
+### compileCSS
 
 Generates a stylesheet for a given style object and selector.
 
 _Parameters_
 
--   _style_ `Style`: Style object.
+-   _style_ `Style`: Style object, for example, the value of a block's attributes.style object or the top level styles in theme.json
```

Comment:
To clarify, does `top level styles in theme.json` mean the whole `styles` object, or is it just the styles that aren't block-specific?

Reply:
Exactly that. Also excluding elements. Basically any object that follows the `{ spacing: { ... }, typography: { ... }, color: { ... }, ... }` model